test/infrastructure/docker/internal/docker/loadbalancer.go

```diff
@@ -71,11 +73,12 @@ func NewLoadBalancer(ctx context.Context, cluster *clusterv1.Cluster, dockerClus
 	image := getLoadBalancerImage(dockerCluster)
 
 	return &LoadBalancer{
-		name:      cluster.Name,
-		image:     image,
-		container: container,
-		ipFamily:  ipFamily,
-		lbCreator: &Manager{},
+		name:             cluster.Name,
+		image:            image,
+		container:        container,
+		ipFamily:         ipFamily,
+		lbCreator:        &Manager{},
+		controlPlanePort: dockerCluster.Spec.ControlPlaneEndpoint.Port,
```

Comment:
Am I wrong or this should also impact the port exposed by the load balancer Pod in 
https://github.com/kubernetes-sigs/cluster-api/blob/1d514cbbf4ad204095e3babb9cff7478f42556f4/test/infrastructure/docker/internal/docker/kind_manager.go#L124

Reply:
you are right, thanks! I will add the change there too.